cli/dcoscli/tables.py

```diff
@@ -423,6 +427,8 @@ def pod_table(pods):
 
     :param pods: pods to render
     :type pods: [dict]
+    :param queued_apps: currently queued apps
+    :type queued_apps: []
```

Comment:
Is this comment correct or is the signature correct? `queue_apps` is not part of the function.